db/seeds.rb

```diff
@@ -1,66 +1,73 @@
+   Item.create(id: 16, title: "Dedenne", description: "desc", price: 1300, image: "http://assets22.pokemon.com/assets/cms2/img/pokedex/full/702.png").categories << electric
+   Item.create(id: 17, title: "Jolteon", description: "desc", price: 1150, image: "http://cdn.bulbagarden.net/upload/thumb/b/bb/135Jolteon.png/600px-135Jolteon.png").categories << electric
+   Item.create(id: 18, title: "Electrode", description: "desc", price: 900, image: "http://cdn.bulbagarden.net/upload/thumb/8/84/101Electrode.png/600px-101Electrode.png").categories << electric
+   Item.create(id: 19, title: "Mew", description: "desc", price: 950, image: "http://cdn.bulbagarden.net/upload/thumb/b/b1/151Mew.png/600px-151Mew.png").categories << psychic
+   Item.create(id: 20, title: "Abra", description: "desc", price: 1000, image: "http://cdn.bulbagarden.net/upload/thumb/6/62/063Abra.png/600px-063Abra.png").categories << psychic
+ # end
 
- def item
-   {"Pikachu"   => "http://core.dawnolmo.com/50_pokemon__9_pikachu_by_megbeth-d5fga3f_original.png",
-   "Charmander" => "http://img4.wikia.nocookie.net/__cb20140603214902/pokemon/images/9/96/004Charmander_OS_anime.png",
-   "Bulbasaur"  => "http://vignette4.wikia.nocookie.net/gpxplus/images/b/b4/001_Bulbasaur_Pic.png/revision/20101225095459",
-   "Squirtle"   => "http://img4.wikia.nocookie.net/__cb20140916184226/pokemon/images/e/e7/007Squirtle_OS_anime.png",
-   "Ivysaur"    => "http://th01.deviantart.net/fs70/PRE/f/2014/002/d/e/pokemon_y__ivysaur_by_smiley_fakemon-d70i6v4.png",
-   "Blastoise"  => "http://fc03.deviantart.net/fs70/f/2010/243/4/c/blastoise_by_xous54-d2xqh0g.png",
-   "Metapod"    => "http://upload.wikimedia.org/wikipedia/id/archive/6/6b/20130416085037!Metapod.png",
-   "Caterpie"   => "http://img3.wikia.nocookie.net/__cb20140911042209/pokemon/images/8/89/010Caterpie_Dream.png",
-   "Pidgeot"    => "http://fc03.deviantart.net/fs70/i/2012/072/6/b/_018_tauboss___pidgeot_by_mrsjasminhardy-d4smgtl.png",
-   "Fearow"     => "http://vignette1.wikia.nocookie.net/es.pokemon/images/7/7b/Fearow_(anime_AG).png/revision/latest?cb=20120906023254",
-   "Sandshrew"  => "http://cdn.bulbagarden.net/upload/9/9e/027Sandshrew.png",
-   "Clefairy"   => "http://img1.wikia.nocookie.net/__cb20140910051252/pokemon/images/e/e8/035Clefairy_Dream.png",
-   "Oddish"     => "http://fc00.deviantart.net/fs70/i/2011/123/9/1/oddish_by_beyx-d3fjesv.png",
-   "Diglett"    => "http://fc06.deviantart.net/fs70/f/2010/165/9/4/050___Diglett_by_cammarin.png",
-   "Meowth"     => "http://images.wikia.com/es.pokemon/images/1/13/Meowth_en_Pok%C3%A9mon_Mundo_Misterioso_2.png",
-   "Dedenne"    => "http://assets22.pokemon.com/assets/cms2/img/pokedex/full/702.png"
-   }
+   o1 = Order.create(status: "ordered", user_id: 1).purchases.create(item_id: 1, quantity:2)
+   o2 = Order.create(status: "completed", user_id: 1).purchases.create(item_id: 2, quantity:1)
+   o3 = Order.create(status: "cancelled", user_id: 1).purchases.create(item_id: 3, quantity:2)
```

Comment:
Useless assignment to variable - `o3`.<br>Line is too long. [94/80]<br>Space missing after colon.